### DISCUSSION TITLE
Various versions of Inter Var

### DISCUSSION THREAD

**Discussion by diligiant** [rsms/inter#558] [Q&A] [answered]
Sorry in advance if that's a rookie question:

I came to discover Inter in latest [tailwindcss releases notes](https://tailwindcss.com/blog/tailwindcss-v3-3#configure-font-variation-settings-for-custom-font-families) and in their example, they are using "Inter Var".

I'm inclined to think that it refers to `Inter.var.woff2` & `Inter-Italic.var.woff2` in 4.0b `Web` folder but I'd love to be sure.

If that's so, does it mean that `Variable` is `Desktop Variable` in its own folder?

Thank you.

**Comment by rsms** [comment, marked as answer]
Hello. It seems to me that the "tailwindcss releases notes" shows an example. On the web platform you choose your font family name in the CSS `@import` statement (so you can call it "lolcat" or "Inter var" or whatever you like; you then must use that exact same name in use sites like `font-family` declarations.)

If you use `https://rsms.me/inter/inter.css` then it is set to `Inter var` for the variable font and `Inter` for the traditional constant fonts.

Now, the zip archive for version 4 contains constant font files for non-web use in the `Desktop` directory and non-web use variable fonts in the `Variable` directory. Web fonts are in the `Web` directory; here you find both constant and variable fonts since the web platform uses both.

> I'm inclined to think that it refers to Inter.var.woff2 & Inter-Italic.var.woff2 in 4.0b Web folder but I'd love to be sure.

That's correct.

> If that's so, does it mean that `Variable` is `Desktop Variable` in its own folder?

I'm not sure what you mean with this. Are you asking if the `Variable` directory contains web fonts? (see above; it does not)
